documentation/general/design/sdn-features-packet-transforms.md

```diff
@@ -303,13 +279,12 @@ For information about packets, see **[Packet direction flow and transforms](sdn-
 
 - 20.0.0.0/24
 
-**VNET Mapping Table**
-| | V4 underlay| V6 underlay| Mac-Address| Mapping Action | VNI
-|:----------|:----------|:----------|:----------|:----------|:----------
-| 10.0.0.1| 100.0.0.1| 3ffe :: 1| Mac1| VXLAN_ENCAP_WITH_DMAC_DE-WRITE| 100
-| 10.0.0.2| 100.0.0.2| 3ffe :: 2| Mac2| VXLAN_ENCAP_WITH_DMAC_DE-WRITE| 200
-| 10.0.0.3| 100.0.0.3| 3ffe :: 3| Mac3| VXLAN_ENCAP_WITH_DMAC_DE-WRITE| 300
-| | | | | |
+**VNET Mapping Table** | | V4 underlay| V6 underlay| Mac-Address| Mapping Action
```

Comment:
This table shows up as unformatted, or formatted strangely

Reply:
Fixed format